### PR TITLE
Documentatino for tabular format

### PR DESCRIPTION
**Pull request by jihyeonyi**
### Summary
Add documentation for tabular data format.

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md).​
- [x] I have updated the [documentation](https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly

### License

- [ ] I submit _my code changes_ under the same [MIT License](https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: docs/source/docs/data-formats/formats/tabular.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+It's because other formats can be converted to `.csv` easily as shown below.
+
+```python
+# convert '.arff' to '.csv'
+from scipy.io.arff import loadarff
+import pandas as pd
+data = loadarff("dataset.arff")
+df = pd.DataFrame(data[0])
+categorical = [col for col in df.columns if df[col].dtype=="O"]
+df[categorical]=df[categorical_columns].apply(lambda x: x.str.decode('utf8'))
```
Comment (vinnamkim) [inline docs/source/docs/data-formats/formats/tabular.md]:
```suggestion
df[categorical] = df[categorical_columns].apply(lambda x: x.str.decode('utf8'))
```

File: docs/source/docs/data-formats/formats/tabular.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+It is also possible to import the dataset using Python API:
+
+```python
+import datumaro as dm
+dataset = dm.Dataset.import_from('<path/to/dataset>', 'tabular')
+```
+
+Datumaro stores the imported table as media (a list of `TableRow`) and annotates the target columns.
+The last column is regarded as the target column,
+which can be specified by the user when when importing the dataset as shown below.
```
Comment (vinnamkim) [inline docs/source/docs/data-formats/formats/tabular.md]:
```suggestion
which can be specified by the user when importing the dataset as shown below.
```

File: docs/source/docs/data-formats/formats/tabular.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+datum project import --format tabular <path/to/electricity/dataset> -- --target class
+```
+
+```python
+import datumaro as dm
+dataset = dm.Dataset.import_from('<path/to/buddy/dataset>', 'tabular', target=["breed_category", "pet_category"])
+dataset = dm.Dataset.import_from('<path/to/electricity/dataset>', 'tabular', target="class")
+```
+
+As shown, the target can be a single column name or a list of columns.
```
Comment (vinnamkim) [inline docs/source/docs/data-formats/formats/tabular.md]:
```suggestion
As shown, the target can be a single column name or a comma-separated list of columns.
```

File: docs/source/docs/data-formats/formats/tabular.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
+# Tabular
+
+## Format specification
+
+Tabular dataset generally referes to table data with multiple rows and columns. <br>
```
Comment (sooahleex) [inline docs/source/docs/data-formats/formats/tabular.md]:
```suggestion
Tabular dataset generally refers to table data with multiple rows and columns. <br>
```

File: docs/source/docs/data-formats/formats/tabular.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+datum project create
+datum project import --format tabular <path/to/buddy/dataset> -- --target breed_category,pet_category
+datum project import --format tabular <path/to/electricity/dataset> -- --target class
+```
+
+```python
+import datumaro as dm
+dataset = dm.Dataset.import_from('<path/to/buddy/dataset>', 'tabular', target=["breed_category", "pet_category"])
+dataset = dm.Dataset.import_from('<path/to/electricity/dataset>', 'tabular', target="class")
+```
```
Comment (sooahleex) [inline docs/source/docs/data-formats/formats/tabular.md:77]: How about showing an example of how a table is displayed?
like `df.head()` in pandas or 
```
|   date   | day | period   | nswprice | nswdemand | vicprice | vicdemand | transfer | class |
|----------|-----|----------|----------|-----------|----------|-----------|----------|-------|
| 0.425556 |  5  | 0.340426 | 0.076108 |  0.392889 | 0.003467 |  0.422915 | 0.414912 |   UP  |
```
If it is not proper this, please ignore this.

Comment (jihyeonyi) [reply docs/source/docs/data-formats/formats/tabular.md:77]: That's a good idea, and I added examples at last.

**Review by sooahleex — APPROVED** [review]
Looks good to me.

**Review by wonjuleee — APPROVED** [review]

**Pull request merged**